applications/GeoMechanicsApplication/tests/cpp_tests/test_dgeosettlement.cpp

```diff
@@ -83,8 +85,9 @@ KRATOS_TEST_CASE_IN_SUITE(RunStageMakesRelevantCallsOnce, KratosGeoMechanicsFast
                                    std::make_unique<StubProcessInfoParser>(),
                                    std::make_unique<StubTimeLoopExecutor>());
 
-    RunStage(settlement);
+    const auto exit_status = RunStage(settlement);
 
+    KRATOS_EXPECT_EQ(exit_status, 1); // Ask Richard why we should expect an error state here
```

Comment:
This is probably because the parameter_json_settings (see line 84) don't have all info needed for the solving strategy factory, so I expect that will throw now. We can either leave it with an explanation, create another interface + stub for the factory, or provide all necessary inputs

Reply:
Done